src/udev/ata_id/ata_id.c

```diff
@@ -615,6 +617,10 @@ static int run(int argc, char *argv[]) {
                 if (IN_SET(identify.wyde[0], 0x848a, 0x844a) ||
                     (identify.wyde[83] & 0xc004) == 0x4004)
                         printf("ID_ATA_CFA=1\n");
+
+                if (peripheral_device_type != -1) {
+                        printf("ID_PERIPHERAL_DEV_TYPE=%d\n", peripheral_device_type);
```

Comment:
Please prefix `ID_ATA_`, and I think it is better to not abbreviate 'DEV'.
That is, `"ID_ATA_PERIPHERAL_DEVICE_TYPE=%d\n"`.